tutorials/nservicebus-101/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "NServiceBus 101: Messaging Basics"
+suppressRelated: true
+---
+
+Welcome to NServiceBus 101! In this self-paced course, you will learn the most important fundamentals of how to get working quickly with NServiceBus.
+
+
+## Objectives
+
+In this course we'll build a simple messaging system from scratch using a retail e-commerce domain.
```

Comment:
remove "simple". either it is clear from the steps that it is simple or it is not simple